v21.2/security-reference/encryption.md

```diff
@@ -13,11 +13,153 @@ CockroachDB uses either TLS 1.2 or TLS 1.3 for inter-node and client-node [authe
 
 ## Encryption at rest
 
-Industry standard encryption-at-rest provided at the infrastructure level by your chosen infrastructure-as-a-service (IAAS) provider, either Google Cloud Platform (GCP) or Amazon Web Services (AWS). See documentation for [GCP persistent disk encryption](https://cloud.google.com/compute/docs/disks#pd_encryption) or [AWS Elastic Block Storage](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSEncryption.html).
+When selecting a deployment environment for a CockroachDB cluster, we recommend that you select a cloud provider such as Google Cloud Platform (GCP), Amazon Web Services (AWS) or Microsoft Azure, which automatically provide industry-standard encryption for cloud storage resources. You can learn more about [GCP persistent disk encryption](https://cloud.google.com/compute/docs/disks#pd_encryption), [AWS EBS volume encryption](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSEncryption.html) or [Azure managed disk encryption](https://docs.microsoft.com/en-us/azure/virtual-machines/disk-encryption-overview). {{ site.data.products.db }} clusters, which can be deployed in either GCP or AWS, automatically receive this protection using cloud provider-managed keys
```

Comment:
CockroachDB dedicated does not support Azure at this time. Is this recommendation meant for only self-hosted customers? I see CMEK (which _is_ a CC-dedicated feature) below.

Reply:
Yes, this is about encryption in CockroachDB period. @abhinavg6 asked that we include info about Azure encryption for self-hosted clusters on Azure VMs.